spec/namespaces/indices.yaml

```diff
@@ -2743,6 +2759,22 @@ components:
             required:
               - _all
               - _shards
+    indices.tier_hot_to_warm@200:
+      description: Hot to warm tiering response
+      content:
+        application/json:
+          schema:
+            type: object
+            properties:
```

Comment:
Take the time to add `description` to all the properties. These will hopefully be used to generate docs soon.

Reply:
added